core/src/main/java/org/apache/accumulo/core/spi/balancer/HostRegexTableLoadBalancer.java

```diff
@@ -153,7 +153,7 @@ static class HrtlbConf {
     private final Map<String,Pattern> poolNameToRegexPattern;
 
     HrtlbConf(PluginEnvironment.Configuration conf) {
-      System.out.println("building hrtlb conf");
+      LOG.info("Building conf");
```

Comment:
@ddanielr It looks like this was just a comment that was added during development. It likely doesn't have much value anymore. At the very least, I'd set it to trace, not info. Info is too high for this, and it is too sparse on details to be very useful as-is.

Reply:
On second thought, a lot of the other log messages in here are INFO. So, this one is probably fine, too.